ethereum/core/src/main/java/tech/pegasys/teku/core/ForkChoiceUtil.java

```diff
@@ -209,43 +209,31 @@ public static void on_tick(MutableStore store, UInt64 time) {
     }
   }
 
-  /**
-   * @param store
-   * @param signed_block
-   * @param maybePreState
-   * @param st
-   * @param forkChoiceStrategy
-   * @param beaconStateConsumer
-   * @see
-   *     <a>https://github.com/ethereum/eth2.0-specs/blob/v0.8.1/specs/core/0_fork-choice.md#on_block</a>
-   */
   @CheckReturnValue
   public static BlockImportResult on_block(
       final MutableStore store,
       final SignedBeaconBlock signed_block,
-      Optional<BeaconState> maybePreState,
+      BeaconState blockSlotState,
```

Comment:
Probably worth some comments on what we're expecting for this state param

Reply:
Added.